client/components/pages/configuration/users/UsersTable.vue

```diff
@@ -67,12 +67,32 @@ function openRowContextMenu(event: Event) {
 <template>
   <UsersTableContextMenu />
 
-  <div class="p-card flex flex-row items-center gap-7 w-full overflow-hidden">
+  <div class="p-card flex flex-row items-center gap-7 w-full h-full max-h-100em overflow-hidden">
+    <div v-if="error" class="h-full w-full flex flex-col justify-center items-center">
+      <!-- TODO: Error state illustration -->
+      <img class="w-10em" :src="useEmptyState('user_list_error').value" alt="Error file">
```

Comment:
avoid using composables directly in the template, because contrary to the setup function, it's ran on every render